9.4/Dockerfile

```diff
@@ -16,14 +16,25 @@ ENV POSTGRESQL_VERSION=9.4 \
     HOME=/var/lib/pgsql \
     PGUSER=postgres
 
-LABEL io.k8s.description="PostgreSQL is an advanced Object-Relational database management system" \
+ENV SUMMARY="Advanced Object-Relational database management system" \
```

Comment:
Is the previous wording `PostgreSQL is an advanced Object-Relational database management system` ok, then?